sunpy/io/tests/test_cdf.py

```diff
@@ -1,16 +1,20 @@
+import datetime as dt
```

Comment:
Why do we need this?

Reply:
this is needed to extract trange in 2nd test function.

Reply:
Why not use strings?

Reply:
Done